#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros_utils'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("extract_num", int_t, 0, "Set the number of extract points", 400, 0, 10000)
gen.add("use_mask_region", bool_t, 0, "Use interactive mask region definition", True)
gen.add("in_the_order_of_depth", bool_t, 0, "extract points in the order of depth", True)
exit (gen.generate (PACKAGE, "jsk_pcl_ros_utils", "MaskImageToDepthConsideredMaskImage"))
